documentation/docs/30-advanced/25-errors.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Errors
+---
+
+Errors are an inevitable fact of software development. SvelteKit handles errors differently depending on where they occur, what kind of errors they are, and the nature of the incoming request.
+
+### Error objects
+
+SvelteKit doesn't expose errors to users directly, Instead, errors are represented as simple `{ message: string }` objects by default.
```

Comment:
I'm guessing the errors talked about here are from `throw new Error` statements, so maybe add "thrown" to clarify (vs let's say a 500 which is an error but not `Error`). Also, there's a comma here that I think should be a period
```suggestion
SvelteKit doesn't expose thrown errors to users directly. Instead, errors are represented as simple `{ message: string }` objects by default.
```

Reply:
Trying to square the circle:

```suggestion
SvelteKit distinguishes between two kinds of errors - expected and unexpected ones. They are explained in more detail in the following sections. Both are represented as simple `{ message: string }` objects by default.
```

Reply:
Yeah I like that. Slightly more compact version — I think the 'Expected errors' heading is close enough that we can omit the 'explained below' bit:

```suggestion
SvelteKit distinguishes between expected and unexpected errors, both of which are represented as simple `{ message: string }` objects by default.
```